crates/sui-rpc-benchmark/src/direct/query_generator.rs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use tokio_postgres::NoTls;
+
+pub struct QueryGenerator {
+    pub db_url: String,
+}
+
+#[derive(Debug, Clone)]
+pub struct BenchmarkQuery {
+    pub query_template: String,
+    pub table_name: String,
+    pub needed_columns: Vec<String>,
+}
+
+impl QueryGenerator {
+    async fn get_tables_and_indexes(&self) -> Result<Vec<BenchmarkQuery>, anyhow::Error> {
+        let (client, connection) = tokio_postgres::connect(&self.db_url, NoTls).await?;
```

Comment:
What's the ratonale for using `tokio_postgres` here instead of `diesel`? It seems like it's another interface for us to get used to.

Reply:
a couple of reasons: 
- tokio_postgres is sufficient while has fewer dependencies, diesel's advantages on query building, migrations etc. are not applicable here
- tokio_postgres, as well as bb8 and bb8-postgres is fairly easy to use, here it's just 2 loc